docs/03-Metrics/hubble_metrics.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I don't think this is working as intended 
![image](https://github.com/user-attachments/assets/ad89b44d-109b-4f9c-be4b-6f6434e18bdb)